### PR TITLE
Include Steps option in aws_emr_cluster

### LINKED ISSUE #7: How do I enable debug logging in this module
As described in https://www.terraform.io/docs/providers/aws/r/emr_cluster.html#enable-debug-logging , Debug logging in EMR is implemented as a step.

### PR DESCRIPTION
**Pull request by hannahkamundson**
## what
You should be able to add steps to this module.

## why
It was initially requested to allow debugging, but it should allow a user to add any steps that they can in the main AWS EMR Terraform 

## references
closes #7

### CODE REVIEW

**Comment by hannahkamundson** [comment]
@RothAndrew @SweetOps I updated this in a fork of this repo and wanted to see if you wanted to include it in your code

File: main.tf [hunk 1]
```diff
@@ -445,6 +445,20 @@ resource "aws_emr_cluster" "default" {
...
   }
 
+  dynamic "step" {
+    for_each = var.steps
+    content {
+      name                                = step.value.name
+      action_on_failure                   = step.value.action_on_failure
+      hadoop_jar_step {
+        jar                               = step.value.hadoop_jar_step["jar"]
+        main_class                        = step.value.hadoop_jar_step["main_class"]
```
Comment (SweetOps) [inline main.tf:455]: Can you add something like this `lookup(step.value.hadoop_jar_step, "main_class", null)` for non-required params?

**Review by SweetOps — CHANGES_REQUESTED** [review]
And pls make PR title more readable... just describe in two words what has been changed and no need reference to `issue` in title

File: main.tf (hunk 1 above)
Comment (hannahkamundson) [reply main.tf]: @SweetOps changed it to lookup + changed the issue title

**Review by SweetOps — APPROVED** [review]

**Pull request merged**